Framework/Algorithms/src/MaxEnt.cpp

```diff
@@ -397,15 +409,17 @@ void MaxEnt::exec() {
       double currAngle = maxentCalculator.getAngle();
       evolChi[it] = currChisq;
       evolTest[it] = currAngle;
-      if (it > maxIt) {
-        maxIt = it;
+      if (it > maxItCount) {
+        maxItCount = it;
       }
+
       // Stop condition, solution found
       if ((std::abs(currChisq / ChiTargetOverN - 1.) < chiEps) &&
           (currAngle < angle)) {
 
-        g_log.information() << "Stopped after " << it << " iterations"
+        g_log.information() << "Stopped after " << it + 1 << " iterations"
```

Comment:
why has this got a plus one now?

Reply:
This is because `it` is counted from zero, one needs to add 1 to get the correct number of iterations.

Reply:
ok

Reply:
@KarlPalmen could you add a comment above this line with your explanation above